app/controllers/application_controller.rb

```diff
@@ -93,6 +93,19 @@ def invalid_recaptcha_score?
     recaptcha_reply["score"] < SUSPICIOUS_RECAPTCHA_THRESHOLD
   end
 
+  def strip_nested_param_whitespaces(object)
```

Comment:
Not sure about this name. I want to be explicit but not confusing. `string_string_whitespaces`?